app/views/mail/jobSuccessfulGeneric.scala.html

```diff
@@ -0,0 +1,29 @@
+@(name: String, datasetName: String, jobLink: String, jobTitle: String, jobDescription: String)
+
+@emailBaseTemplate() {
+<p>Hi @{name},</p>
+
+<p>We wanted to let you know that WEBKNOSSOS has finished your background job for the dataset <i>@{datasetName}</i>. 
```

Comment:
"We wanted to let you know that" could be removed in my opinion as it doesn't add much value.